scripts/actions/add-files-to-translation-queue.js

```diff
@@ -52,10 +72,11 @@ const getLocalizedFileData = (prFile) => {
   const contents = fs.readFileSync(path.join(process.cwd(), prFile.filename));
   const { data } = frontmatter(contents);
   const checkLocale = getProjectId(data.translate);
+  const fileType = data.type;
```

Comment:
this is just a musing that i have, not necessarily something to address since i think this would be a bit of effort. so you can just leave it 🙈 

we should rename this to something like `contentType` or `pageType` since file type has a meaning already and we are also using it to represent the team defined type in graphql.

Reply:
cool ye that makes sense